src/components/Dashboard.vue

```diff
@@ -186,6 +186,19 @@
         const duration = this.$root.MomentJS.duration(parseFloat(decimalTime)).asMilliseconds();
         return this.$root.MomentJS().startOf('day').add(duration, 'minutes').format('m:ss');
       },
+      finishTime() {
```

Comment:
As mentioned, it should be a general function (put them in `utils` folder).

Should get the car, socDisplay, socBMS, batteryPower (maybe instead just all the syncData).

The other usage should then also be fetched from the utils.

Reply:
Ok, done. Please have a look on my commit 8c96624. Added the file and new general function. Changed the functions calls in dashboard.vue.

Reply:
Wow, thank you :+1: :tada: 

Will test it later and look at it :)